src/V3Timing.cpp

```diff
@@ -614,25 +614,12 @@ class TimingVisitor final : public VNVisitor {
             VL_RESTORER(m_procp);
             m_procp = beginp;
             iterate(beginp);
-            if (!m_procp->user2()) {
-                // No awaits, we can inline this process
-                if (auto* const stmtsp = beginp->stmtsp()) {
-                    nodep->addHereThisAsNext(stmtsp->unlinkFrBackWithNext());
-                }
-                VL_DO_DANGLING(beginp->unlinkFrBack()->deleteTree(), beginp);
-                // We inlined at least one process, so we can consider it joined; convert join_any
-                // to join_none
-                if (nodep->joinType().joinAny()) nodep->joinType(VJoinType::JOIN_NONE);
-            } else {
-                // Name the begin (later the name will be used for a new function)
-                beginp->name(nodep->name() + "__" + cvtToStr(idx++));
-            }
-        }
-        if (!nodep->stmtsp()) {
-            VL_DO_DANGLING(nodep->unlinkFrBack()->deleteTree(), nodep);
```

Comment:
Where do we remove empty forks now? Probably need a test for it too to make sure we don't crash on dereferencing a nullptr somewhere.

Reply:
Empty forks get removed early on in `V3Width`.